RELEASES.md

```diff
@@ -1,3 +1,117 @@
+
+<a id="1.84.0-Compiler"></a>
+
+Compiler
+--------
+- [Add `--print host-tuple` flag to print the host target tuple and affirm the "target tuple" terminology over "target triple"](https://github.com/rust-lang/rust/pull/125579)
+- [Declaring functions with a calling convention not supported on the current target now triggers a hard error](https://github.com/rust-lang/rust/pull/129935)
+- [Set up indirect access to external data for `loongarch64-unknown-linux-{musl,ohos}`](https://github.com/rust-lang/rust/pull/131583)
+- [Enable XRay instrumentation for LoongArch Linux targets](https://github.com/rust-lang/rust/pull/131818)
+- [Extend the `unexpected_cfgs` lint to also warn in external macros](https://github.com/rust-lang/rust/pull/132577)
+- [Stabilize WebAssembly `multivalue`, `reference-types`, and `tail-call` target features](https://github.com/rust-lang/rust/pull/131080)
+- [Execute drop glue when unwinding in an `extern "C"` function](https://github.com/rust-lang/rust/issues/130076)
+
+<a id="1.84.0-Libraries"></a>
+
+Libraries
+---------
+- [Implement `From<&mut {slice}>` for `Box/Rc/Arc<{slice}>`](https://github.com/rust-lang/rust/pull/129329)
+- [Move `<float>::copysign`, `<float>::abs`, `<float>::signum` to `core`](https://github.com/rust-lang/rust/pull/131304)
+- [Add `LowerExp` and `UpperExp` implementations to `NonZero`](https://github.com/rust-lang/rust/pull/131377)
+- [Implement `FromStr` for `CString` and `TryFrom<CString>` for `String`](https://github.com/rust-lang/rust/pull/130608)
+- [`std::os::darwin` has been made public](https://github.com/rust-lang/rust/issues/131161)
+
+<a id="1.84.0-Stabilized-APIs"></a>
+
+Stabilized APIs
+---------------
+
+- [Ipv6Addr::is_unique_local]
+- [Ipv6Addr::is_unicast_link_local]
+- [`core::ptr::with_exposed_provenance`]
+- [`core::ptr::with_exposed_provenance::mut`]
+- [`<ptr>::addr`]
+- [`<ptr>::expose_provenance`]
+- [`<ptr>::with_addr`]
+- [`<ptr>::map_addr`]
+
+These APIs are now stable in const contexts
+
+- [`AtomicBool::from_ptr]
+- [`AtomicPtr::from_ptr`]
+- [`AtomicU8::from_ptr`]
+- [`AtomicU16::from_ptr`]
+- [`AtomicU32::from_ptr`]
+- [`AtomicU64::from_ptr`]
+- [`AtomicUsize::from_ptr`]
+- [`AtomicI8::from_ptr`]
+- [`AtomicI16::from_ptr`]
+- [`AtomicI32::from_ptr`]
+- [`AtomicI64::from_ptr`]
+- [`AtomicIsize::from_ptr`]
+- [`<ptr>::is_null`]
+- [`<ptr>::as_ref`]
+- [`<ptr>::as_mut`]
+- [`core::ptr::without_provenance`]
+- [`core::ptr::without_provenance_mut`]
+- [`core::ptr::dangling`]
+- [`core::ptr::dangling_mut`]
+- [`Pin::new`]
+- [`Pin::new_unchecked`]
+- [`Pin::get_ref`]
+- [`Pin::into_ref`]
+- [`Pin::get_mut`]
+- [`Pin::get_unchecked_mut`]
+- [`Pin::static_ref`]
+- [`Pin::static_mut`]
+- [`<int>::isqrt`]
+- [`<int>::checked_isqrt`]
+- [`<uint>::isqrt`]
+- [`NonZero::isqrt`]
+
+<a id="1.84.0-Cargo"></a>
+
+Cargo
+-----
+- [Stabilize MSRV-aware resolver config](https://github.com/rust-lang/cargo/pull/14639/)
```

Comment:
I think this one may be worth a mention in the blog post, to give guidance to the ecosystem on what it means for them.